modules/backend/lang/hu/lang.php

```diff
@@ -24,14 +24,14 @@
         ],
         'access_denied' => [
             'label' => 'Hozzáférés megtagadva',
-            'help' => 'Ön nem rendelkezik a szükséges engedélyekkel ennek a lapnak a megtekintéséhez.',
+            'help' => 'Nem rendelkezik a szükséges engedélyekkel ennek a lapnak a megtekintéséhez.',
             'cms_link' => 'Vissza a látogatói oldalra'
         ],
         'no_database' => [
             'label' => 'Az adatbázis nem elérhető',
             'help' => 'Kérjük ellenőrizze a hozzáférési adatok helyességét majd próbálja újra betölteni az oldalt.',
             'cms_link' => 'Vissza a weboldalra'
-        ],
+        ]
```

Comment:
All array elements should have a trailing comma in the language files, this makes it much easier to update them in the future. I'm not going to force people to add them but I do draw the line at them being removed @gergo85 😄 (@w20k @bennothommo FYI)

Reply:
Alright. Next time I will put comma everywhere :)